adr/001-update-pfs-message.md

```diff
@@ -0,0 +1,107 @@
+# `UpdatePFS` message design
+
+* Status: accepted
+* Deciders: Augusto, Dominik, Karl, Konrad, Paul
+* Date: 2019-04-09
+
+## Context and Problem Statement
+
+The PFS requires information about the capacity and mediation fees of channels from Raiden nodes. This information must be passed on by the Raiden nodes. The design of the balance and fee updates must optimize for two contradicting paradigms
+
+1. Accuracy of the global state from the PFS's perspective
+2. Privacy of the Raiden nodes
+
+The nodes can decide themselves whether to send balance and fee updates to the PFS to advertise their channel for mediating or not. 
+
+## Decision Drivers
+
+* Channel capacities are computable
+* Privacy for nodes
+* Behaviour with cheating nodes
+
+## Considered Options
+
+1. Option 1: A solution where each participant broadcasts the signed, clear balance proofs received from the other participant to the PFS' via the gloabl matrix room.
+2. Option 2: A solution where the message contains the capacities of both channel sides and both participants broadcast them to the PFS' via the gloabl matrix room. Conflicting statements will be resolved by taking the mininma of the provided values.
+
+## Decision Outcome
+
+Chosen option: "Option 2", because it allows for more privacy while offering only smaller safe capacities in the worst case.
+
+### Positive Consequences
+
+* Message incluides only capacity, so the balance proofs stay private
+* Privacy is possible because possibility to broadcast only at certain thresholds and not for every update
+
+### Negative Consequences
+
+* In case where one of the channel participants is malicious, the available capacities will be computed lower than in option 1.
+
+## Detailed analysis
+
+To look at the feasibility of both design one needs to look at three different scenarios for the channel participants:
+1. Both participants are well behaving.
+2. Both participants are malicious.
+3. One of the participants is malicious.
+
+### Case 1: Two well behaving participants
+
+In this case the PFS learns about the correct channel capacities in both message designs.
+
+### Case 2: Two malicious participants
+
+In this case the PFS can not do anything to find out about the malicious behaviour. However, we assume that not all participants in the network are malicious.
+Therefore there must exist a channel, where case 3 is active.
```

Comment:
German word order. I suggest "Therefore, a channel must exit where case 3 is active".

Reply:
nice formulation